lib/shared/src/sourcegraph-api/graphql/client.ts

```diff
@@ -1249,7 +1250,16 @@ export class SourcegraphGraphQLAPIClient {
         }
         return {}
     }
-
+    // Deletes an access token, if it exists on the server
+    public async DeleteAccessToken(token: string): Promise<unknown | Error> {
+        const initialResponse = await this.fetchSourcegraphAPI<APIResponse<unknown>>(
+            DELETE_ACCESS_TOKEN_MUTATION,
+            {
+                token,
+            }
+        )
+        return extractDataOrError(initialResponse, data => data)
+    }
```

Comment:
```suggestion
    }
    
```

Let's separate these methods with one blank line for consistency.